instrumentation/opentelemetry-instrumentation-fastapi/src/opentelemetry/instrumentation/fastapi/__init__.py

```diff
@@ -172,10 +172,10 @@ def client_response_hook(span: Span, scope: dict[str, Any], message: dict[str, A
 ---
 """
 import logging
-from importlib.util import find_spec
 from typing import Collection
 
 import fastapi
+from pkg_resources import get_distribution
```

Comment:
Not a big deal but is it a lot of work to use `importlib.metadata` instead since that is encouraged over `pkg_resources`